schemas/core/components/configurator.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Configurator schema to customize a booking option",
+  "type": "object",
+  "description": "Wrapper schema, with each of its properties a config that the user can customize",
+  "patternProperties": {
+    "^[a-z\\-]+$": {
+      "$ref": "#/definitions/config"
+    }
+  },
+  "definitions": {
+    "config": {
+      "type": "object",
+      "description": "A customization to the booking option",
+      "properties": {
+        "type": {
+          "description": "Describe the possible combination of choices user can make",
+          "enum": ["oneOf", "someOf", "allOf", "oneOrNoneOf", "someOrNoneOf"]
```

Comment:
What would be a use case for `allOf`? Wouldn't it mean no real choice?

Reply:
Yeah, if all the choices are included, then we can have them shown to the client purely for information purpose. E.g with Sixt, `extra driver` and `snow package` are included to every option